### DISCUSSION TITLE
Is BC v1.0.2.3 FIPS compliant on JDK17?

### DISCUSSION THREAD

**Discussion by canquillo09** [bcgit/bc-java#1436] [Q&A] [answered]
Hi,

It is not clear on the official site whether if BouncyCastle is FIPS compliant on Java services based on JDK 17. I only found information that "BC-FJA 2.0.0 was submitted for certification under FIPS 140-3 on Feb 3rd, 2022" here -> https://www.bouncycastle.org/fips_java_roadmap.html. 

Thanks in advance!

**Comment by dghgit** [comment]
Only 1.0.2.4 is compliant with Java 17. The module is currently making it's way through the submission queue.

**Reply by canquillo09** [reply]
Hi @dghgit ,

So is BouncyCastle FIPS compliant on Java services based on JDK 17?

**Reply by dghgit** [reply, marked as answer]
Only the 1.0.2.4 version is compliant on Java 17. The 2.0.0 release will also be released with Java 17 as a tested environment.

**Reply by canquillo09** [reply]
Thank you very much @dghgit! 👍
